README.md

```diff
@@ -1,32 +1,51 @@
 Overview
 ========
 
-Token SDK for iOS. The SDK is built on top of protobuf gRPC API definition.
+Token SDK for iOS. Docs https://developer.token.io/sdk/
 
-Pick up new protos
-------------------
+The SDK is built on top of protobuf gRPC API definition,
+https://developer.token.io/sdk/pbdoc/io_token_proto_gateway.html
+
+Client Usage
+------------
+
+To use the SDK version compatible with Token's Sandbox testing
+environment (typical usage), use the published pod:
+
+```
+  pod 'TokenSdk', '~> 1.0.72'
+```
+
+The SDK can be added to a client directly from git.
+
+```
+  pod 'TokenSdk',  :git => 'https://github.com/tokenio/sdk-objc',:submodules => true
+```
 
-For now the protos are just copied over from `../lib-proto`. The script fails if the directory is not found. To pick up new protos run `./bin/update-protos`. After the script finished executing one needs to reload the updated workspace file (with XCode or IntelliJ AppCode).
+or referenced locally in a Podfile
+
+```
+  pod 'TokenSdk', :path => '../..'
+```
```

Comment:
for external users, we will not suggest them to use local TokenSdk pods. (pod update will not update local pods)

I am also hesitating if we need to keep the submodule option, since I might only be available against dev env.